mappings/net/minecraft/sortme/Explosion.mapping

```diff
@@ -1 +1,46 @@
 CLASS none/aip net/minecraft/sortme/Explosion
+	FIELD a createFire Z
+	FIELD b destroyBlocks Z
+	FIELD c rand Ljava/util/Random;
+	FIELD d world Lnone/aiu;
+	FIELD e x D
+	FIELD f y D
+	FIELD g z D
+	FIELD h entity Lnone/se;
+	FIELD i power F
+	FIELD j affectedBlocks Ljava/util/List;
+	FIELD k affectedPlayers Ljava/util/Map;
+	METHOD <init> (Lnone/aiu;Lnone/se;DDDFLjava/util/List;)V
+		ARG 0 world
+		ARG 1 entity
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+		ARG 5 power
+		ARG 6 affectedBlocks
+	METHOD <init> (Lnone/aiu;Lnone/se;DDDFZZ)V
+		ARG 0 world
+		ARG 1 entity
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+		ARG 5 power
+		ARG 6 createFire
+		ARG 7 destroyBlocks
+	METHOD <init> (Lnone/aiu;Lnone/se;DDDFZZLjava/util/List;)V
+		ARG 0 world
+		ARG 1 entity
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+		ARG 5 power
+		ARG 6 createFire
+		ARG 7 destroyBlocks
+		ARG 8 affectedBlocks
+	METHOD a collectBlocksAndDamageEntities ()V
```

Comment:
This is one weird name but it's basically what this function does. It collects all blocks that would be destroyed and then damages and pushes around entities. `affectWorld` then actually destroys these blocks, creates particles and fire.